.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    parserOptions: {
+      ecmaVersion: 2019,
+      sourceType: "module",
+    },
+    env: {
+      browser: true,
+      es6: true,
+    },
+    extends: ["airbnb-base"],
+    plugins: ["jest", "svelte3"],
+    rules: {
+      "import/prefer-default-export": "off",
+      "import/no-extraneous-dependencies": ["error", { devDependencies: true }],
+      "no-param-reassign": [
+        "error",
+        { props: true, ignorePropertyModificationsFor: ["draft"] },
+      ],
+    },
```

Comment:
Do these rules need to be modified? I actually prefer these, but this file is also going to end up in repos that are created using this template. I'm wary of creating a new standard that developers need to keep track of: the airbnb/graph-paper standard. It may be easier to use the standard airbnb rules.

Alternatively, we could tell degit to ignore this file with a degit configuration file.

Reply:
I wouldn't overthink this too much. This template is for us ( / originally me), not for others. They can change whatever they want going forward to meet their needs – in fact, I would expect it.